extension/persistence/eclipselink/src/main/resources/META-INF/persistence.xml

```diff
@@ -34,8 +34,7 @@
     <class>org.apache.polaris.jpa.models.ModelSequenceId</class>
     <shared-cache-mode>NONE</shared-cache-mode>
     <properties>
-      <property name="jakarta.persistence.jdbc.url"
-        value="jdbc:h2:file:./build/test_data/polaris/{realm}/db"/>
```

Comment:
Why is this changing?

Reply:
Because I don't see the value of creating a file on disk? 
That said, if you really insist on having that file created, I can revert this change.

Reply:
I agree - using in-memory here is better.